### PR TITLE
fix: also take perProtocolMappings into subtype options

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by kelsos** [comment]
@lukicenturi can you provide a small explanation of how to test this change?

What can I select and what should I expect?

Event better is it possible to add some test for this?

**Comment by lukicenturi** [comment]
> @lukicenturi can you provide a small explanation of how to test this change?
> 
> What can I select and what should I expect?
> 
> Event better is it possible to add some test for this?

For a brief explanation, if you go to history events form
Then you set the eventType to `deposit` you will see 4 `eventSubTypes` options.
If then you choose protocol `yearn-v2` you will see  additional `receive wrapped` in the `eventSubTypes` options.

It's because there are eventSubtypes that only exists on `perProtocolMappings` but not in `globalMappings`.
Will add the test

**Comment by kelsos** [comment]
> > @lukicenturi can you provide a small explanation of how to test this change?
> > What can I select and what should I expect?
> > Event better is it possible to add some test for this?
> 
> For a brief explanation, if you go to history events form Then you set the eventType to `deposit` you will see 4 `eventSubTypes` options. If then you choose protocol `yearn-v2` you will see additional `receive wrapped` in the `eventSubTypes` options.
> 
> It's because there are eventSubtypes that only exists on `perProtocolMappings` but not in `globalMappings`. Will add the test

Great thank you I will give it a try then to see how things work, thanks for the details.

File: frontend/app/tests/unit/fixtures/history-event-counterparties.json [hunk 1]
```diff
@@ -0,0 +1,220 @@
+{
+  "result": [
```
Comment (kelsos) [inline frontend/app/tests/unit/fixtures/history-event-counterparties.json:2]: Do you think we can reduce the content of the fixtures to the bare minimum that is required to test the functionality? This looks a bit like the full response from the API, we certainly don't need everything.

**Review by kelsos — APPROVED** [review]
looks good @lukicenturi thank you

**Pull request merged**